src/poetry/core/masonry/builders/wheel.py

```diff
@@ -361,13 +361,14 @@ def _get_sys_tags(self) -> list[str]:
 """,
                 ],
                 stderr=subprocess.STDOUT,
+                text=True,
             )
         except subprocess.CalledProcessError as e:
             raise RuntimeError(
                 "Failed to get sys_tags for python interpreter"
                 f" '{self.executable.as_posix()}':\n{decode(e.output)}"
```

Comment:
If we pass `text=True` `e.output` will be decoded, too. Then we can remove `poetry.core.utils.helpers.decode` altogether since it was only introduced for this part in the last patch release.

Reply:
it wasn't clear to me whether that was true or not - some of the subprocess errors say "This is always bytes when any output was captured regardless of the text=True setting" and some of them just... don't say that.

Reply:
OK, that was only an observation on my part. If we can't count on that, you are right to do the `decode()`.

Reply:
however experiment seems to confirm that this one is indeed a string